examples/graphql-scalar-id/graphql_id_scalar.md

```diff
@@ -0,0 +1,23 @@
+# GraphQL service - ID Scalar Type
```

Comment:
```suggestion
# GraphQL service - ID scalar type
```

Reply:
Fixed in [73bf589](https://github.com/ballerina-platform/ballerina-distribution/pull/5188/commits/73bf589184b148f87e83d105386f7ec2710527b7)